### PR TITLE
German Translation of Getting Started

### LINKED ISSUE #6963: German translation of Getting started
- [ ] Join the Free Code Camp community
 - [ ] Learn how Free Code Camp Works 
 - [ ] Create a GitHub Account and Join our Chat Rooms
 - [ ] Configure your Code Portfolio
 - [ ] Join a Campsite in Your City
 - [ ] Learn What to Do If You Get Stuck

### PR DESCRIPTION
**Pull request by abisz**
German translation of the 00-getting-started challenges

closes #6963

### CODE REVIEW

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -234,6 +234,81 @@
...
+        [
+          "//i.imgur.com/vLNso6h.jpg",
+          "Eine Illustration die zeigt dass du HTML5, CSS3, JavaScript, Datenbanken, Git, Node.js, React und D3 lernen wirst",
+          "Wir haben hunderte von optionalen Programmieraufgaben die dir die fundamentalen Webentwicklungstechnologien wie HTML5, Node.js und Datanbanken beibringen werden.",
+          ""
+        ],
+        [
+          "//i.imgur.com/UVB9hxp.jpg",
+          "Ein Foto von einem Camper in einem Cafe der an Free Code Camp Projekten arbeitet.",
+          "Wir glauben dass Menschen am Besten beim Tun lernen. Dewegen wirst du die meiste Zeit damit beschäftigt sein an tatsächlichen Projekten zu arbeiten. Wir werden dir eine Liste an Anforderungen (agile User Stories) geben und du wirst dich damit auseinander setzen diese zu erfüllen.",
```
Comment (vtamara) [inline seed/challenges/00-getting-started/getting-started.json]: Dewegen -> Deswegen

**Comment by vtamara** [comment]
Good job @abisz 
Long ago I didn't read german, but I think it is good translation.

Please make the little improvement I suggested, to keep only one commit message, after improving your branch (add/DE-translation-GettingStarted)  better use
```
git commit --amend
git -f push add/DE-translation-GettingStarted
```

Nex time, please call your branch ```translate/DE-GettingStarted``` or similar starting with ```translate/```

I wish a german native speaker could check this.  Could you please ask in the chat room TranslationDeutsch reviewing of this pull request?  If no one helps, I think this can be merged, but for future translations to german other reviewer will be required.

**Comment by abisz** [comment]
I posted it into the chat room, but I doubt somebody will look into it (due to the fact that there are only 4 members - including us)... I think I'll look for other german-speaking people in the other rooms for future translations. 

But I have another question. Yesterday, after pulling, I found a german translation of other challenges and noticed that the translator didn't translate "challenge", which I did. I actually prefer not translating "challenge", should I change that?

And one last question :) if I do another commit to this branch, will it automatically update this pull request or do I have to make another one? I'm not quite sure about how GitHub works regarding this issue...

**Comment by abisz** [comment]
Ok now I messed up, I guess I'll close this PR and make a new one, @vtamara? Or is there any way I can fix this?

**Comment by raisedadead** [comment]
@abisz Yes you can. Just [squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one)

**Comment by raisedadead** [comment]
No need to close the PR.

**Comment by abisz** [comment]
@raisedadead thanks for the help, it was a hard struggle, but I think I made it work

**Comment by vtamara** [comment]
Good job. Regarding translation of challenge, the idea would be to keep a glossary. And there to keep record how to translate special words.   I suggest you to use https://fcc-dict.herokuapp.com/login

**Comment by vtamara** [comment]
For the moment I merged.  Please discuss translation of challenge in the chat room and if it is decided that better leave untraslated send a new pull request for that change (can be along with translation to a different file :) )

**Pull request merged**